README.md

```diff
@@ -0,0 +1 @@
+![image](https://user-images.githubusercontent.com/91699763/166444077-613cefef-6149-4cb2-b132-3df8c52f1292.png)
```

Comment:
Card: where is the FK for user?
Remove underlines for tables that handles many-to-many

Reply:
resolved

Reply:
Looks good, one think to note: mark Ids as a FK where it is needed

Reply:
resolved